core/jobs/batch_jobs/rejecting_suggestion_for_invalid_content_ids_jobs.py

```diff
@@ -0,0 +1,586 @@
+                        collapsible_content,
+                        is_tags_nested_inside_tabs_or_collapsible=True)
+                )
+                if (
+                    RejectSuggestionWithMissingContentIdMigrationJob.
+                    _is_tag_removed_with_empty_content(
+                        tag, collapsible_content, is_collapsible=True)
+                ):
+                    continue
+
+                collapsible_content_json = json.dumps(collapsible_content)
+                tag['content-with-value'] = utils.escape_html(
+                    collapsible_content_json)
+
+            else:
+                tag.decompose()
+                continue
+
+            if (
+                RejectSuggestionWithMissingContentIdMigrationJob.
+                _is_tag_removed_with_invalid_attributes(
+                    tag, 'heading-with-value')
+            ):
+                continue
+
+        return str(soup).replace('<br/>', '<br>')
+
+    @staticmethod
+    def _fix_content(html: str) -> str:
+        """Helper function to fix the html.
+
+        Args:
+            html: str. The html data to fix.
+
+        Returns:
+            html: str. The fixed html data.
+        """
+        html = RejectSuggestionWithMissingContentIdMigrationJob.fix_rte_tags(
+            html, is_tags_nested_inside_tabs_or_collapsible=False)
+        html = (
+            RejectSuggestionWithMissingContentIdMigrationJob.
+            fix_tabs_and_collapsible_tags(html))
+        return html
+
+    @staticmethod
+    def _update_suggestion_model(
+        suggestions: List[suggestion_models.GeneralSuggestionModel],
+        exp_model: exp_models.ExplorationModel
+    ) -> None:
+        """Updates the translation suggestion. The translation whose
+        content_id no longer exists, the suggestion status will be marked
+        as `rejected`. The RTE content of the suggestion will be updated
+        in case invalid data is present.
+
+        Args:
+            suggestions: list(GeneralSuggestionModel). A list of translation
+                suggestion models corresponding to the given exploration.
+            exp_model: ExplorationModel. The exploration model.
+
+        Returns:
+            suggestions. List[GeneralSuggestionModel]. Result containing the
+            list of updated suggestion models.
+        """
+        exp_domain = exp_fetchers.get_exploration_from_model(exp_model)
+        exp_translatable_contents = (
+            exp_domain.get_translatable_contents_collection())
+
+        translatable_content_ids = []
+        for content_id in (
+            exp_translatable_contents.content_id_to_translatable_content.keys()
+        ):
+            translatable_content_ids.append(content_id)
+
+        for suggestion in suggestions:
+            suggestion_change = suggestion.change_cmd
+            if not suggestion_change['content_id'] in translatable_content_ids:
+                suggestion.status = 'rejected'
```

Comment:
There should exist constant for `'rejected'`

Reply:
Updated